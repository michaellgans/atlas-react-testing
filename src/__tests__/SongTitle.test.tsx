// Test for SongTitle Component

// Imports
import { render } from "@testing-library/react";
import { expect, test } from "vitest";
import { SongTitle } from "@/components/SongTitle";
import { ContextProvider } from "@/components/ContextProvider";

test("SongTitle renders correctly", () => {
    const {asFragment} = render(
        <ContextProvider>
            <SongTitle />
        </ContextProvider>
    );
    expect(asFragment).toMatchSnapshot();
});
