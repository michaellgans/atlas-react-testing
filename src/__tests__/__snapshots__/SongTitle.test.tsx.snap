// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SongTitle renders correctly 1`] = `[Function]`;
